include/hse/hse.h

```diff
@@ -169,6 +169,26 @@ hse_param_get(const char *param, char *buf, size_t buf_sz, size_t *needed_sz);
  * @{
  */
 
+/** @brief Attach a new KVDB to a snapshot/copy of the media class paths from kvdb_home_src.
+ *
+ * @note This function is not thread safe.
+ *
+ * @param kvdb_home_tgt: KVDB home directory for the attach target.
+ * @param kvdb_home_src: KVDB home directory for the attach source.
+ * @param paths: a snapshot/copy of the media class paths from the attach source.
+ *
+ * @remark @p kvdb_home_tgt and kvdb_home_src must not be NULL.
+ * @remark kvdb_home_src must be offline for attach
```

Comment:
Add an <at> p before kvdb_home_src

Reply:
This doc stuff is a lot of overhead, does anyone look at it?

Reply:
I do in order make sure it looks valid and consistent.